train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java

```diff
@@ -1,13 +1,26 @@
 package hu.bme.mit.train.controller;
 
 import hu.bme.mit.train.interfaces.TrainController;
+import java.util.Timer;
+import java.util.*;
```

Comment:
Importing java.util.TimerTask should be enough

Reply:
Replaced * with TimerTask